src/lib/datepicker/datepicker.md

```diff
@@ -228,3 +228,6 @@ application root module.
 })
 export class MyApp {}
 ```
+### Accessibility
+The input box for datepicker should have a placeholder or be given a meaningful label via
```

Comment:
I'd like to expand this to describe what the datepicker does in general (input with aria-haspopup that triggers a `role="dialog"`) along with the keyboard shortcuts, but that can go in another PR.